src/server/pps/server/worker_rc.go

```diff
@@ -262,10 +262,15 @@ func (a *apiServer) getWorkerOptions(pipelineName string, rcName string,
 		imagePullSecrets = append(imagePullSecrets, api.LocalObjectReference{Name: secret})
 	}
 
+	annotations := map[string]string{"pipelineName": pipelineName}
+	if a.iamRole != "" {
+		annotations["iam.amazonaws.com/role"] = a.iamRole
+	}
```

Comment:
What is this annotation used for? I don't see it being read anywhere.

Reply:
The pods need the annotation so they can assume the node's IAM role.  That's just how k8s on AWS works.  It's not really directly used by our code.